src/android/CameraLauncher.java

```diff
@@ -39,7 +39,7 @@ Licensed to the Apache Software Foundation (ASF) under one
 import android.os.Bundle;
 import android.os.Environment;
 import android.provider.MediaStore;
-import android.support.v4.content.FileProvider;
+import androidx.core.content.FileProvider;
```

Comment:
this requires something like `implementation "androidx.core:core:$CORE_VERSION"` in the user apps, I can't find it in cordova-android, so I guess the plugin should still have the framework entry in plugin.xml and make the version configurable